### PR TITLE
Issue 191 - Multi-state visualization

### PR DESCRIPTION
**Pull request by NatHillardUSDS**
Addresses #191 - "As a stakeholder interested in the cumulative impact score, I want to see more states in the map, so that I can further analyze the score results."

Will need further visual tweaks but this works with current data

### CODE REVIEW

File: client/src/components/map.tsx [hunk 1]
```diff
@@ -20,17 +21,25 @@ const mapConfig = {
...
+        'https://c.basemaps.cartocdn.com/light_nolabels/{z}/{x}/{y}.png',
+        'https://d.basemaps.cartocdn.com/light_nolabels/{z}/{x}/{y}.png',
       ],
     },
     'custom': {
-      'projection': 'EPSG:3857',
       'type': 'vector',
       'tiles': [
-        'https://gis.data.census.gov/arcgis/rest/services/Hosted/VT_2019_150_00_PY_D1/VectorTileServer/tile/{z}/{y}/{x}.mvt',
+        'http://localhost:8080/data/tl_2010_bg_with_data/{z}/{x}/{y}.pbf',
```
Comment (TomNUSDS) [inline client/src/components/map.tsx:33]: `localhost` url. Maybe add a `todo` to fix it?

File: client/src/components/map.tsx [hunk 2]
```diff
@@ -44,50 +53,100 @@ const mapConfig = {
...
+    React.MutableRefObject<HTMLInputElement>;
+  const mapRef = useRef() as React.MutableRefObject<Map>;
+
+
+  const popupContainer = React.useRef<HTMLDivElement>(null);
+  const popupCloser = React.useRef<HTMLAnchorElement>(null);
+  const popupContent = React.useRef<HTMLDivElement>(null);
+
+  const overlayRef = useRef() as React.MutableRefObject<Overlay>;
+  mapRef.current = map!;
```
Comment (TomNUSDS) [inline client/src/components/map.tsx:114]: I'm not familiar with `map!` in this context. What is it doing?

File: client/src/components/map.tsx [hunk 3]
```diff
@@ -112,8 +171,105 @@ const MapWrapper = ({features}: IMapWrapperProps) => {
...
+  let properties;
+  if (selectedFeature) {
+    properties = selectedFeature.getProperties();
+  }
+
+  const getCategorization = (percentile: number) => {
+    let categorization = '';
+    if (percentile > 0.75 ) {
+      categorization = 'Prioritized';
+    } else if (0.60 < percentile && percentile < 0.75) {
```
Comment (TomNUSDS) [inline client/src/components/map.tsx:205]: nice choice for formatting. 
Should one of these choices be >= ? (e.g. what if percentile is exactly 0.75)

**Review by TomNUSDS — COMMENTED** [review]
Looks great. Think I spotted on possible boundary condition?

File: client/src/components/map.tsx (hunk 3 above)
Comment (NatHillardUSDS) [reply client/src/components/map.tsx:205]: Good catch! will update

File: client/src/components/map.tsx (hunk 1 above)
Comment (NatHillardUSDS) [reply client/src/components/map.tsx]: Good call - in the latest push I use a remote url and add the localhost as a comment

File: client/src/components/map.tsx (hunk 2 above)
Comment (NatHillardUSDS) [reply client/src/components/map.tsx:116]: this is the non-null assertion operator ([ref](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator)) - you occasionally have to use this in Swift and I was surprised to find it in Typescript also - basically it's assuring the compiler that you know better and that this thing actually does exist.

That said, while this particular usage had the effect of temporarily silencing the compiler, in this case it actually might not exist, so this should be handled differently. In general we should move away from using `ref`s in this way - it works but there are better ways to do this.

Comment (NatHillardUSDS) [reply client/src/components/map.tsx:116]: I can look into other solutions and push a spot fix for this

Comment (TomNUSDS) [reply client/src/components/map.tsx:116]: I've never seen it used to stop linter/compiler complaints this way. I'm fine with it with maybe a comment or `todo`.

**Review by TomNUSDS — APPROVED** [review]
LGTM!

**Pull request merged**